### ISSUE TITLE
modelEvalDiscrete.m is not in the repo

### ISSUE THREAD

**Issue by char4816** [jonathanhorrocks/SINDy-data#1]
Hello, we are unable to run your code because modelEvalDiscrete.m is not in the repo.  Might you please be able to add it?  

Thank you,
Chris

**Comment by Geethani** [comment]
Yes, Could you please add the file, as we couldn't run your code.
